.github/workflows/cd-syft.yml

```diff
@@ -129,17 +129,17 @@ jobs:
           cache-from: type=registry,ref=openmined/grid-backend:cache-${{ steps.release_metadata.outputs.short_release_platform }}
           cache-to: type=registry,ref=openmined/grid-backend:cache-${{ steps.release_metadata.outputs.short_release_platform }},mode=max
 
-      - name: Build and push `syft-base-cpu` image to DockerHub
-        id: syft-base-cpu-build
-        uses: docker/build-push-action@v5
-        with:
-          context: ./packages
-          file: ./packages/grid/backend/syft_base_cpu.dockerfile
-          platforms: ${{ steps.release_metadata.outputs.release_platform }}
-          target: backend
-          outputs: type=image,name=openmined/syft-base-cpu,push-by-digest=true,name-canonical=true,push=true
-          cache-from: type=registry,ref=openmined/syft-base-cpu:cache-${{ steps.release_metadata.outputs.short_release_platform }}
-          cache-to: type=registry,ref=openmined/syft-base-cpu:cache-${{ steps.release_metadata.outputs.short_release_platform }},mode=max
+      # - name: Build and push `syft-base-cpu` image to DockerHub
+      #   id: syft-base-cpu-build
+      #   uses: docker/build-push-action@v5
+      #   with:
+      #     context: ./packages
+      #     file: ./packages/grid/backend/worker_cpu.dockerfile
+      #     platforms: ${{ steps.release_metadata.outputs.release_platform }}
+      #     target: backend
+      #     outputs: type=image,name=openmined/syft-base-cpu,push-by-digest=true,name-canonical=true,push=true
+      #     cache-from: type=registry,ref=openmined/syft-base-cpu:cache-${{ steps.release_metadata.outputs.short_release_platform }}
+      #     cache-to: type=registry,ref=openmined/syft-base-cpu:cache-${{ steps.release_metadata.outputs.short_release_platform }},mode=max
```

Comment:
If the `worker_cpu` image is always meant to be built inside a backend container in prod and not prebuilt and pushed to DockerHub then maybe we can just delete this instead of commenting it out?

Reply:
I think we will shift to having a base image with just the syft code in it. And the workers can later build from that. But if I think in that case as well maybe backend builds from the base image and worker builds from backend. Anyways I think we are just keeping this commented for bookkeeping, but we can choose to delete this, @yashgorana thoughts ?

Reply:
Yeah for now we can keep this. we can housekeep at the towards the end of the helm feature.